src/Sylius/Bundle/CustomerBundle/Resources/translations/messages.fr.yml

```
@@ -6,19 +6,19 @@ sylius:
         customer:
             first_name: Prénom
             group: Groupe
-            last_name: Nom
-            email: Courriel
-            phone_number: Téléphone
-            gender: Sexe
-            birthday: Date de naissance
+            last_name: Nom de famille
+            email: Adresse électronique
+            phone_number: Numéro de téléphone
+            gender: Genre
+            birthday: Date d'anniversaire
```

```suggestion
            last_name: Nom
            email: Courriel
            phone_number: Téléphone
            gender: Sexe
            birthday: Date de naissance
```

Please revert this change as the original translations are more accurate